example/config/tech-docs.yml

```diff
@@ -52,3 +52,11 @@ api_path: source/pets.yml
 # Optional global settings for the page review process
 owner_slack_workspace: gds
 default_owner_slack: '#2nd-line'
+
+# cookie_banner: 
+#   enable: true
+#   cookie_types: ['essential','analytics', 'additional']
+#   essential_cookies_text: 'Essential cookies'
+#   analytics_cookies_text: 'Analytics cookies'
+#   additional_cookies_text: 'Additional cookies'
```

Comment:
Would it be easier (and match more what the GOV.UK Design System component does) to have one variable for setting text rather than a differently named variable depending on the cookie types?

Reply:
easier: probably no different
match DS component logic: yes

Reply:
It might help to simplify the code and might make it easier for users to understand how to override the message. E.g: if we need to show a custom message because we're implementing sub-domain consent, it becomes less obvious which string to override when we just need a whole new message.

Reply:
update to one variable and if present replaces default content.
there are still conditionals in places for when this text is not provided